### PR TITLE
Urlscan Batch

### PR DESCRIPTION
**Pull request by avidan-H**
## Status
In Progress

## Related Issues
fixes: https://github.com/demisto/etc/issues/19727 https://github.com/demisto/etc/issues/19801

## Description
Adds batch support to the urlscan integration `url` command. Relevant for this [issue](https://github.com/demisto/etc/issues/19495).

## Related PRs
List related PRs against other branches:

branch | PR
------ | ------
ipinfo-batch | https://github.com/demisto/content/pull/4699

## Required version of Demisto
any

## Does it break backward compatibility?
   - No

## Must have
- [x] Tests
- [ ] Documentation (with link to it)
- [ ] Code Review

## Dependencies
Mention the dependencies of the entity you changed as given from the precommit hooks in checkboxes, and tick after tested them.
- [x] UrlscanGetHttpTransactions
- [x] urlscan_malicious_Test

## Additional changes
Fixed parsing of content that is no longer returned by the API call used in the `url` command. Updated test playbook.

## Technical writer review
Mention and link to the files that require a technical writer review.
- [ ] [YAML file](https://github.com/demisto/content/edit/urlscan-batch/Integrations/UrlScan/URLScan.yml?pr=%2Fdemisto%2Fcontent%2Fpull%2F4703)
- [ ] [CHANGELOG](https://github.com/demisto/content/edit/urlscan-batch/Integrations/UrlScan/URLScan_CHANGELOG.md?pr=%2Fdemisto%2Fcontent%2Fpull%2F4703)

### CODE REVIEW

File: Integrations/UrlScan/URLScan.py [hunk 1]
```diff
@@ -272,13 +272,6 @@ def format_results(uuid):
             dbot_score['Score'] = 0
             dbot_score['Type'] = 'url'
             human_readable['Malicious'] = 'Benign'
-    if 'url' in scan_meta['processors']['gsb']['data'] is None:
```
Comment (Itay4) [inline Integrations/UrlScan/URLScan.py]: we are sure it is no longer returned?

File: Integrations/UrlScan/URLScan.py [hunk 2]
```diff
@@ -357,7 +350,11 @@ def get_urlscan_submit_results_polling(uuid):
 
 
 def urlscan_submit_command():
-    get_urlscan_submit_results_polling(urlscan_submit_url())
+    urls = argToList(demisto.args().get('url'))
+    for url in urls:
+        demisto.args()['url'] = url
```
Comment (Itay4) [inline Integrations/UrlScan/URLScan.py]: this is quite ugly.. can we refactor the code a bit or is it too much ?

File: Integrations/UrlScan/URLScan.py (hunk 1 above)
Comment (avidan-H) [reply Integrations/UrlScan/URLScan.py]: @amshamah419 said that it was a breaking change in the API over the weekend but I can confirm if you'd like me to doublecheck

**Comment by amshamah419** [comment]
Screen Shot 2019-10-29 at 10 37 31

FYI

**Comment by kirbles19** [comment]
@avidan-H MD reviewed.

**Comment by YuvHayun** [comment]
@avidan-H  In this PR, you deprecated the following commands
urlscan-poll-uri which is being used in: Packs/UrlScan/Scripts/UrlscanGetHttpTransactions/UrlscanGetHttpTransactions.yml
urlscan-submit-url-command which is being used in: 
Packs/UrlScan/Scripts/UrlscanGetHttpTransactions/UrlscanGetHttpTransactions.yml
urlscan-get-http-transaction-list which is being used in: 
Packs/UrlScan/Scripts/UrlscanGetHttpTransactions/UrlscanGetHttpTransactions.yml

Can you take a look at that and try to replace the mentioned commands in those places / if there's anything to do with that script?

**Pull request merged**